lib/git/aliases.sh

```diff
@@ -17,7 +17,7 @@ unalias git > /dev/null 2>&1
 unset -f git > /dev/null 2>&1
 
 # Use the full path to git to avoid infinite loop with git function
-export _git_cmd="$(\which git)"
+export _git_cmd="$(bin_path git)"
```

Comment:
Alternatively, one could use `command -v git` instead.

Reply:
Retrieve a binary pathname, not a function or alias.